ext/travis/test

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "$TRAVIS_OS_NAME" = linux ]; then
```

Comment:
I think we might want to refactor this to have a travis script which calls a script that can be shared by jenkins and travis.